### PR TITLE
Single sign on login view

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Style login view for SSO. It is used if `oidc_enabled` is set to true in the configuration.

image

## How was this tested?

Jest

### CODE REVIEW

File: assets/js/pages/Login/LoginSSO.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Button from '@common/Button';
+
+export default function LoginSSO({ singleSignOnUrl }) {
+  return (
+    <Button
+      onClick={() => {
+        window.location.href = singleSignOnUrl;
```
Comment (CDimonaco) [inline assets/js/pages/Login/LoginSSO.jsx:9]: We are not using a child href for styling purposes?

**Review by CDimonaco — APPROVED** [review]
LGTM 🚀

**Review by jagabomb — APPROVED** [review]
LGTM, (reviewed using the screenshot)

File: assets/js/pages/Login/LoginSSO.jsx (hunk 1 above)
Comment (arbulu89) [reply assets/js/pages/Login/LoginSSO.jsx:9]: You mean to have `<a href={ singleSignOnUrl }>Login with Single Sign-on</a>` as child, right?
This has the difference that the url is visible in the browser, when you put the mouse on the button.
I don't have any preference. I have seen the `window.location.href` in other login/session react component in our code. That's why I used it

**Review by EMaksy — APPROVED** [review]
Hey the code lgtm!
I guess you took a wip from @CDimonaco and the unused imports variables and console logs will go away inn a follow up pr right ?

**Comment by arbulu89** [comment]
> Hey the code lgtm! I guess you took a wip from @CDimonaco and the unused imports variables and console logs will go away inn a follow up pr right ?

Yes, fix coming in next PR

File: assets/js/pages/Login/LoginSSO.jsx (hunk 1 above)
Comment (arbulu89) [reply assets/js/pages/Login/LoginSSO.jsx:9]: And yes, the button is what we need related to styling, so it was an "obvious" option

**Pull request merged**